src/components/TreesMap/3d/trees-3d-layer.ts

```diff
@@ -0,0 +1,185 @@
+import mapboxgl from 'mapbox-gl';
+
+export const trees3DHighlightSourceId = 'trees-3d-highlight-source';
+export const trees3DHighlightLayerId = 'trees-3d-highlight-layer';
+export const trees3DCylinderSourceId = 'trees-3d-cylinders-source';
+export const trees3DCylinderLayerId = 'trees-3d-cylinders-layer';
+export const trees3DLayerId = 'trees-3d-layer';
+export const treeModelId = 'tree-general-model';
+
+const MAPBOX_TREE_CYLINDERS_TILESET_URL =
+  process.env.NEXT_PUBLIC_MAPBOX_TREE_CYLINDERS_TILESET_URL;
+const MAPBOX_TREE_CYLINDERS_LAYERNAME =
+  process.env.NEXT_PUBLIC_MAPBOX_TREE_CYLINDERS_LAYERNAME;
+
+export const trees3DLayer = {
+  id: trees3DLayerId,
+  //@ts-ignore
```

Comment:
Why so many `//@ts-ignore`

Reply:
because there are no types available yet for a lot of the 3d layer features we are using here